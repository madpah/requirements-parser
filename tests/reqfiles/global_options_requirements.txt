-i https://pypi.org/simple
-c
-f /my/local/archives
--find-links /my/local/archives
--no-binary :none:
--only-binary :all:
--prefer-binary
--require-hashes

six==1.16.0