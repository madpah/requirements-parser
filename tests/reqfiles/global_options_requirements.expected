[
 {
  "specifier": true,
  "local_file": false,
  "name": "six",
  "editable": false,
  "subdirectory": null,
  "uri": null,
  "extras": [],
  "vcs": null,
  "path": null,
  "line":  "six==1.16.0",
  "hash_name": null,
  "hash": null,
  "specs": [
    [
    "==",
    "1.16.0"
   ]
  ],
  "revision": null
 }
]